recipes/tests.rb

```diff
@@ -208,11 +208,19 @@
 # Amazon Time Sync
 ###################
 if node['init_package'] == 'init'
-  chrony_check_command = "service #{node['cfncluster']['chrony']['service']} status | grep -i running"
+  get_chrony_status_command = "service #{node['cfncluster']['chrony']['service']} status"
 elsif node['init_package'] == 'systemd'
   # $ systemctl show -p SubState <service>
   # SubState=Running
-  chrony_check_command = "systemctl show -p SubState #{node['cfncluster']['chrony']['service']} | grep -i running"
+  get_chrony_status_command = "systemctl show -p SubState #{node['cfncluster']['chrony']['service']}"
+end
+chrony_check_command = get_chrony_status_command + " | grep -i running"
+
+ruby_block 'log_chrony_status' do
+  block do
+    chrony_status = shell_out!(get_chrony_status_command).stdout
```

Comment:
this will report the status of the service without giving other info. In case of failure we know from the ` `chrony_check_command` that the status isn't running. Probably would be useful to retrieve the full service log, e.g. for `journalctl -u node['cfncluster']['chrony']['service']`